src/components/UserToken.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { getToken } from '../lib/TokenService';
+
+const UserToken = () => {
+  const handleCopyToken = () => {
+    const copyData = document.getElementById('user-token').innerHTML;
+    navigator.clipboard.writeText(copyData);
+  };
+  return (
+    <div className="user-token">
+      <b>
+        List token: "<span id="user-token">{getToken()}</span>"
+      </b>
+      <button aria-label="Copy token" onClick={handleCopyToken}>
+        <i className="fa fa-clone"></i>
+      </button>
+    </div>
```

Comment:
I love this feature! Very clever of you 💡
One thing I could suggest is to add a 2-3 second `flash message` or just `text` indicating that the token was copied.

Reply:
Thank you, Luka! Yes, that's actually a good idea. :)